imap/utils.go

```diff
@@ -16,7 +18,18 @@ func login(ctx context.Context, d *plugin.QueryData) (*client.Client, error) {
 	port := 993
 	tlsEnabled := true
 	insecureSkipVerify := false
-	var host, login, password string
+
+	// Check env var settings
+	host := os.Getenv("IMAP_HOST")
+	login := os.Getenv("IMAP_LOGIN")
+	password := os.Getenv("IMAP_PASSWORD")
+
+	if portString, ok := os.LookupEnv("IMAP_PORT"); ok {
+		p, err := strconv.Atoi(portString)
+		if err == nil {
```

Comment:
@graza-io I think we should check if they're passing a valid integer and a valid port number, like we have in https://github.com/turbot/steampipe-plugin-aws/blob/ed9e5a8b2b463fb6e30bd2fc19bbb16ab9dc8607/aws/service.go#L1163-L1165. Please let me know your thoughts.